README.md

```diff
@@ -37,19 +38,52 @@ have built thus far for their respective teams.
 
 - __ads-card (vault)__ - Very basic card component, in use in Vault a couple places.
   [(link)](https://git.archive.org/dps/vault-site/-/blob/fe99de1d53c62f426b1d6b70cc100321d651a9aa/frontend/src/web-components/ads-card.ts)
+- __ia-image-card__ - Simple component to display an image as a card, with an optional overlay color, hover-title, and on-click link. Used by various custom collections such as [Democracy's Library](https://archive.org/details/democracys-library). Point: @rebecca-shoptaw. [(link)](https://git.archive.org/www/offshoot/-/blob/main/src/widgets/ia-image-card.ts)
 
 ### Toggle Switch
 
 - __ads-toggle-switch (vault)__ - Nice toggle switch with CSS animation moving toggle back and forth.
   [(link)](https://git.archive.org/dps/vault-site/-/blob/fe99de1d53c62f426b1d6b70cc100321d651a9aa/frontend/src/web-components/ads-toggle-switch.ts)
+- __toggle-switch (UX team)__ - Toggle switch with screen-reader support and a lot of CSS customization options. Point: @nsharma123. [(link)](https://git.archive.org/www/offshoot/-/blob/main/src/widgets/toggle-switch.ts)
```

Comment:
I could be a point person for this one too -- I think the one in Offshoot was largely copied from the original in https://github.com/internetarchive/iaux-collection-browser/blob/main/src/collection-facets/toggle-switch.ts